bin/run-r-scripts

```diff
@@ -1,15 +1,14 @@
 #! /bin/bash
 
-# Set permissions so that new files can be deleted/overwritten outside docker
 umask 000
-
 cd /bound
 
-# Install faster, using binaries for linux from RStudio package manager
-echo 'options(repos = c(CRAN = "https://packagemanager.rstudio.com/all/__linux__/bionic/latest"))' >> ~/.Rprofile
-# Rscript -e "update.packages(ask = FALSE)"
-Rscript -e "install.packages(c('here','writexl','countrycode', 'renv', 'tibble', 'stringr', 'zoo', 'data.table'))"
-# FIXME: TestPortfolio_Input should be a parameter
-Rscript --vanilla web_tool_script_1.R TestPortfolio_Input
-Rscript --vanilla web_tool_script_2.R TestPortfolio_Input
-Rscript --vanilla web_tool_script_3.R TestPortfolio_Input
+# FIXME: Eventually we want to run the Rscripts directly, but it's easier
+# to detect problems if you run them via integration-test.Rmd because some of
+# its tests should allow you to narrow down a bit the cause of the problem
+Rscript -e "rmarkdown::render('preconditions.Rmd', output_file = tempfile())"
```

Comment:
If this is what's meant to run on the web server, I strongly believe that it should not waste any time nor risk potential problems by rendering a markdown file

Reply:
I'm not sure what the problem is with rendering markdown, but it it is an issue we could `knitr::purl()` the .Rmd file to convert it to an .R file, then `source()` it.

In any case, this is likely temporary, just to help us debug.

Reply:
running just that command on my machine takes 15 second or so to run, so it's not conducive to make the web server feel quick and responsive